src/main/resources/init_db.sql

```diff
@@ -0,0 +1,9 @@
+CREATE SCHEMA IF NOT EXISTS home_work;
+
+USE home_work;
+
+CREATE TABLE bookS (
+    id BIGINT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    title VARCHAR(255) NOT NULL ,
+    price DECIMAL(10,2) NOT NULL
```

Comment:
i think 10 is a very big price for a book, it's like 1000000000,00 for a book, but it's your choice